restore-sql-server-transaction-logs/Function/main.py

```diff
@@ -0,0 +1,274 @@
+import functions_framework
+import google.auth
+import google.auth.transport.requests
+import googleapiclient.discovery
+import logging
+import os
+from time import sleep
+from google.cloud import storage
+from google.cloud import error_reporting
+
+
+def get_environment_variables() -> [bool, str, str, str, int, int, int]:
+    
+    err_msg=""
+
+    if not os.environ.get('USE_FIXED_FILE_NAME_FORMAT'):
+        err_msg = "Environment variable USE_FIXED_FILE_NAME_FORMAT is None or empty."
+        return False, err_msg, "", "", 0, 0, 0
+
+    if os.environ.get('USE_FIXED_FILE_NAME_FORMAT').lower() not in ["true", "false"]:
+        err_msg = "Environment variable USE_FIXED_FILE_NAME_FORMAT must be either True or False."
+        return False, err_msg, "", "", 0, 0, 0
+        
+
+    use_fixed_file_name_format = os.environ.get('USE_FIXED_FILE_NAME_FORMAT')

```

Comment:
Use this assignment before any error check, and check the value of the `use_fixed_name_format`. This will reduce duplication. For example, with the current implementation, if you change the name of the `USE_FIXED_FILE_NAME_FORMAT` key, you'll need to change it in 3 places, at least.

Search for the same pattern across this implementation, and fix that as well.

Reply:
LGTM